malformed/test_malformed_headers.py

```diff
@@ -0,0 +1,487 @@
+from framework import tester
+from helpers import tf_cfg, deproxy
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class MalformedRequestsTest(tester.TempestaTest):
+    backends = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'port' : '8000',
+            'response' : 'static',
+            'response_content' :
+"""HTTP/1.1 200 OK
+Content-Length: 0
+Connection: close
+
+"""
+        },
+    ]
+
+    tempesta = {
+        'config' : """
+cache 0;
+listen 80;
+
+srv_group default {
+    server ${general_ip}:8000;
+}
+
+vhost default {
+    proxy_pass default;
+}
+""",
+    }
+
+    clients = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'addr' : "${tempesta_ip}",
+            'port' : '80'
+        },
+    ]
+
+    def common_check(self, request):
+        deproxy = self.get_server('deproxy')
+        deproxy.start()
+        self.start_tempesta()
+        self.assertTrue(deproxy.wait_for_connections(timeout=1))
+        deproxy = self.get_client('deproxy')
+        deproxy.start()
+        deproxy.make_request(request)
+        resp = deproxy.wait_for_response(timeout=5)
+        self.assertTrue(resp, "Response not received")
+        status = deproxy.last_response.status
+        self.assertEqual(int(status), 400, "Wrong status: %s" % status)
+
+    def test_accept(self):
+        # https://tools.ietf.org/html/rfc7231#section-5.3.2
```

Comment:
It's a bad test description. The section can provide a lot of information an different statements. Please describe which exact statement is described in the test. The description must be self-sufficient, RFC link must be interpreted as information source.

This is related to many other tests.

Reply:
fixed